src/signature/mod.rs

```diff
@@ -24,6 +29,51 @@ impl TryFrom<u8> for SignatureVersion {
     }
 }
 
+/// copied from wasm/wasm-crypto
+fn create_from_pair(pair: &[u8]) -> EdKeypair {
+    match EdKeypair::from_bytes(pair) {
+        Ok(pair) => return pair,
+        Err(_) => panic!("Provided pair is invalid."),
+    }
+}
+
+/// copied from wasm/wasm-crypto
+fn create_from_parts(pubkey: &[u8], seckey: &[u8]) -> EdKeypair {
```

Comment:
too many `create_*` functions, hard to follow, would recommend inlining them all into one function for verify and one function for sign